lib/eapeak/inject.py

```diff
@@ -23,28 +23,29 @@
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
 #  MA 02110-1301, USA.
 
-__version__ = '0.0.6'
-
 # native imports
 import Queue
 from random import randint
 from struct import pack, unpack
 import threading
 import time
-import threading
-
-# project imports
-from eapeak.clients import WirelessClient
-from eapeak.common import getBSSID, getSource, getDestination
-from eapeak.networks import WirelessNetwork
-from eapeak.parse import UNKNOWN_SSID_NAME, parseRSNData, buildRSNData
-from ipfunc import getHwAddr
 
 # external imports
 from scapy.layers.dot11 import RadioTap, Dot11, Dot11Beacon, Dot11Elt, Dot11Auth, Dot11AssoReq, Dot11AssoResp, Dot11ProbeReq, Dot11Disas, Dot11QoS, Dot11ProbeResp
 from scapy.layers.l2 import LLC, SNAP, EAPOL
 from scapy.sendrecv import sniff, sendp
-from eapeak.scapylayers.l2 import LEAP, PEAP, EAP
+
+# project imports
+from eapeak.common import get_BSSID, get_source, get_destination
+from eapeak.parse import parse_rsn_data, build_rsn_data
+from eapeak.scapylayers.l2 import LEAP, PEAP, EAP #pylint: disable=unused-import
```

Comment:
For comments, there should be exactly two spaces on the left of the `#` and one on the right.